### DISCUSSION TITLE
Subject: YOLOv8 Support on Raspberry Pi with OpenVINO

### DISCUSSION THREAD

**Discussion by mj-stride** [openvinotoolkit/openvino#26117] [General]
Subject: YOLOv8 Support on Raspberry Pi with OpenVINO

Hi everyone,

I’m currently working with OpenVINO on a Raspberry Pi setup and I'm interested in using YOLOv8. Could someone confirm whether OpenVINO supports YOLOv8 models on this platform? Additionally, if there are any specific instructions or known issues for running YOLOv8 on a Raspberry Pi with OpenVINO, I would appreciate any guidance or tips.

Thanks in advance!

**Comment by alvoron** [comment]
Hello @mj-stride,

YOLO models are supported by OpenVINO on ARM platforms, so you shouldn't expect any issues with running YOLOv8 on RaspberryPi.

If you take RaspberryPi 5 OpenVINO does fp16 inference - it gives you much better performance.
If you have no sample program with yolo you may start with yolov8 OpenVINO notebook:
https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/yolov8-optimization

If you face any issues on RaspberryPi - please don't hesitate to raise an issue in GitHub.
